scripts/data_prep/convert_delta_to_json.py

```diff
@@ -610,7 +631,7 @@ def fetch_DT(args: Namespace) -> None:
     parser.add_argument(
         '--delta_table_name',
         required=True,
-        type=str,
+        type=format_tablename,
```

Comment:
do you like this better than keeping this as string and then running format_table_name in fetch DT?

Reply:
Now running this within `fetch_DT` to make things clearer.